fizzbuzz/fizzbuzz.c

```diff
@@ -10,7 +10,27 @@
 */
 int fizzbuzz(int n)
 {
-
+int count = 0;
+    for (int i = 0; i < n; i++)
+    {
+        if (i % 3 == 0)
+        {
+            printf("Fizz\n");
+        }
+        else if (i % 5 == 0)
+        {
+            printf("Buzz\n");
+        }
+        else if (i % 5 == 0 && i % 3 == 0)
+        {
+            printf("FizzBuzz\n");
+        }
+        else
+        {
+            count++;
+        }
+    };
+    return count;
```

Comment:
This looks good. However, I would start with the case that `if (i % 15 == 0)` and then move to `else if (i % 3 == 0)` and `else if (i % 5 == 0)`. The 15 means the number is divisible by both 5 and 3.